src/Microsoft.Azure.WebJobs.Extensions.Redis/RedisUtilities.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.Azure.WebJobs.Extensions.Redis
 {
-    internal static class RedisUtilities
+    public static class RedisUtilities
```

Comment:
It seems like the other PRs do not need this change. Are you sure it is needed with the current structure? I think if you change the namespace of your files to: `Microsoft.Azure.WebJobs.Extensions.Redis.Samples` it would be accessible because it is set to `internal`.

Reply:
When I change the namespace to `Microsoft.Azure.WebJobs.Extensions.Redis.Samples`, I still get the error that `RedisUtilities` is inaccessible due to its protection level.

Reply:
Oh I see, they are not using it. Sorry for the confusion, you can keep your changes as the are now!